share/sql/alerting.sql

```diff
@@ -2,70 +2,138 @@ SET search_path TO monitoring, public;
 
 BEGIN;
 
-CREATE TYPE check_state_type AS ENUM('OK', 'WARNING', 'CRITICAL', 'UNDEF');
+DROP TYPE IF EXISTS check_state_type CASCADE;
+DROP TYPE IF EXISTS check_results_record CASCADE;
+DROP TABLE IF EXISTS check_results_current;
+DROP TABLE IF EXISTS check_results_history;
+DROP TABLE IF EXISTS check_states;
+DROP TABLE IF EXISTS checks;
 
-CREATE TYPE check_results_record AS (
-	datetime TIMESTAMPTZ,
-	state check_state_type,
-	key VARCHAR(64),
-	value REAL,
-	threshold REAL
-);
+CREATE TYPE check_state_type AS ENUM('OK', 'WARNING', 'CRITICAL', 'UNDEF');
 
 CREATE TABLE checks (
-	check_id SERIAL PRIMARY KEY,
-	host_id INTEGER NOT NULL REFERENCES hosts (host_id),
-	instance_id INTEGER REFERENCES instances (instance_id),
-	enabled BOOLEAN NOT NULL DEFAULT false,
-	name VARCHAR(64) NOT NULL,
-	threshold_w REAL,
-	threshold_c REAL,
-	description TEXT
+  check_id SERIAL PRIMARY KEY,
+  host_id INTEGER NOT NULL REFERENCES hosts (host_id),
+  instance_id INTEGER REFERENCES instances (instance_id),
+  enabled BOOLEAN NOT NULL DEFAULT false,
+  name VARCHAR(64) NOT NULL,
+  threshold_w REAL,
+  threshold_c REAL,
+  description TEXT
```

Comment:
If SQL indentation is 2 spaces, let's configure it in `.editorconfig`.

Reply:
Done with https://github.com/dalibo/temboard/pull/321/commits/0f74c25a02988aaf6fa7eb33f4eea480741ce234